tests/client/components/__snapshots__/carousel.spec.js.snap

```diff
@@ -1,40 +1,42 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think that snapshots are temp files and shouldn't be kept in the repository.

Reply:
Snapshots are temp files, but we need to keep them in repo to compare against on new testing runs. Otherwise they would just get generate every time, and never be able to fail

Reply:
ok, so I'm not going to change this anymore